sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/encoders/RowEncoder.scala

```diff
@@ -92,6 +93,7 @@ object RowEncoder {
         NewInstance(
           classOf[GenericArrayData],
           inputObject :: Nil,
+          propagateNull = true,
```

Comment:
#10443 will fix this by using true as default value for propagateNull. As that pr is not merged yet, in order to pass tests now, I set propagateNull as true here. Once #10443 is getting merged, I can remove the two lines here.